src/lucky_record/model.cr

```diff
@@ -30,7 +30,8 @@ class LuckyRecord::Model
     id.to_s
   end
 
-  macro table(table_name, primary_key_type = LuckyRecord::PrimaryKeyType::Serial)
+  macro table(table_name = nil, primary_key_type = LuckyRecord::PrimaryKeyType::Serial)
+    {% table_name = run("../run_macros/infer_table_name.cr", @type.id) unless table_name %}
```

Comment:
I think the `unless` at the end of this kind of sneaks in and makes it hard to see. Could you move it to above? `{% unless table_name %}`?